package.json

```diff
@@ -40,7 +40,9 @@
   },
   "dependencies": {
     "ansi-to-html": "0.x",
+    "antd": "2.x",
```

Comment:
The previous PR still hardcoded the version. I must have forgotten to include it in the last commit.